api/routes/application/routes.py

```diff
@@ -156,7 +156,10 @@ def put(self):
             "is_summary_page_submit": request_json["metadata"].get("isSummaryPageSubmit", False),
         }
         try:
-            updated_form = update_form(**form_dict)
+            updated_form, round_status = update_form(**form_dict)
+            if not round_status:
```

Comment:
More readable if this was called `is_round_open` or similar

Reply:
Fixed